server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema(
+    {
+        firstname: {
+            type: String,
+            required: true,
+        },
+        lastname: {
+            type: String,
+            required: true,
+        },
+        gender: {
+            type: String,
+            required: true,
+        },
```

Comment:
This would be a nice opportunity to explore some [`string validators`](https://mongoosejs.com/docs/schematypes.html#string-validators), for example we could use an `enum` for all the options we want to support (I recommend including `other` to conform with the modern standards 😄 ), paired with any others that you think could be useful!